modules/opentelemetry-cpp/1.16.0/MODULE.bazel

```diff
@@ -0,0 +1,25 @@
+# Copyright The OpenTelemetry Authors
+# SPDX-License-Identifier: Apache-2.0
+
+module(
+    name = "opentelemetry-cpp",
+    version = "0",
```

Comment:
have to update this to be the actual version, and reflect that with a patch file if the sources at this tag haven't done that (which i assume they havent, since it's another step)

Reply:
PTAL